targets/mgapilinter/targets.go

```diff
@@ -17,23 +17,17 @@ import (
 const version = "1.29.3"
 
 // nolint: gochecknoglobals
-var executable string
+var commandPath string
```

Comment:
If we really want, we should be able to remove this global variable.

I'm thinking of the following:
* Specify constants for binary name, version, etc
* Add a function that builds the path based of those constants and any variables, like path to tools dir

Could be a future improvement.

Reply:
Really like this!

Reply:
Actually, given that we added symlinks to all tools we download, the path is now deterministic. Should be a fairly simple change.

Reply:
However, i will look at that in a separate PR